README.md

```diff
@@ -48,15 +48,27 @@ That things work as the requirements describe. It's okay to ask questions if you
 
 If we like the end result, we will want you to talk us through your code and any decisions you made.
 
-### Running the back end
+## Running the back end
 
-The back end requires Java 8, and can be run from the `backend` directory with the command `./run.sh`
+The back end service can be run from the `backend` directory by following these steps
+1. open a terminal from the `backend` directory
+2. run  `npm ci` to install the dependencies required to run the back end service
+3. after successful packages installation, run `npm run dev`, the starts the back end service.
```

Comment:
Nitpick:
3. after **a** successful packages installation, run `npm run dev` **to start** the back end service. (or something similar)